npm-shrinkwrap.json

```
@@ -13135,7 +13149,7 @@
 			}
 		},
 		"node_modules/npm/node_modules/ip": {
-			"version": "2.0.1",
+			"version": "2.0.0",
```

we need to update it to patch version as the 2.0.0 is marked by dependebot as security threat as it has some well known security issue. As this is not our direct dependency but as sub dependency coming from a different package we need to update it manually everytime we rebuild shrinkwrap

```suggestion
			"version": "2.0.1",
```